### ISSUE TITLE
PNL and B/S reports are not correctly indented for Full hierarchy

### ISSUE THREAD

**Issue by aungzwin10** [ledgersmb/LedgerSMB#5990]
Current indentation as below
<img width="282" alt="Screen Shot 2021-09-16 at 12 11 06 PM" src="https://user-images.githubusercontent.com/31498748/133555987-2b0cf05d-c9a1-4172-9187-056d7234fbc7.png">

It should indent like below, no?
<img width="324" alt="Screen Shot 2021-09-16 at 12 14 56 PM" src="https://user-images.githubusercontent.com/31498748/133556381-5d1987a1-fa7b-42ef-9da5-af1cba63a727.png">

**Comment by ehuelsmann** [comment]
Yes, it should. I'm thinking that the thing missing, is probably the CSS required to do the indentation.

**Comment by aungzwin10** [comment]
No, the CSS for indentation is available, but i think the behavior is not same as you expected. I can inspect in the browser DevTools for the below CSS.
https://github.com/ledgersmb/LedgerSMB/blob/5f8f19c1a911fbf7340538d2cbf96e009523729d/UI/css/system/global.css#L365-L399
